Duplicati/Server/WebServer/RESTMethods/Filesystem.cs

```diff
@@ -208,7 +208,9 @@ private static IEnumerable<Serializable.TreeNode> ListFolderAsNodes(string entry
                 return false;
             };
 
-            foreach (var s in SystemIO.IO_OS.EnumerateFileSystemEntries(entrypath))
+            foreach (var s in SystemIO.IO_OS.EnumerateFileSystemEntries(entrypath)
+                .OrderByDescending(f => SystemIO.IO_OS.GetFileAttributes(f) & FileAttributes.Directory)
+                .ThenBy(f => SystemIO.IO_OS.PathGetFileName(f)))

```

Comment:
Is `SystemIO.IO_OS.PathGetFileName` necessary, or does `.ThenBy(f => f)` achieve the same result?

Reply:
I'll test it and let you know.  If it does still work I'll update the pull request.

Reply:
Simplification works